crates/ruff_python_formatter/src/string/any.rs

```diff
@@ -164,21 +164,30 @@ impl FusedIterator for AnyStringPartsIter<'_> {}
 /// string. This could be either a string, bytes or f-string.
 ///
 /// This is constructed from the [`AnyString::parts`] method on [`AnyString`].
-#[derive(Clone, Debug)]
-pub(super) enum AnyStringPart<'a> {
+#[derive(Clone, Copy, Debug)]
+pub(crate) enum AnyStringPart<'a> {
     String(&'a ast::StringLiteral),
     Bytes(&'a ast::BytesLiteral),
     FString(&'a ast::FString),
 }
```

Comment:
I think we could possibly remove this as well and use the one in the `ruff_python_ast` crate:

https://github.com/astral-sh/ruff/blob/61fc39bdb6dd0da7d594a9d2755a353eceb3a771/crates/ruff_python_ast/src/expression.rs#L446-L452

Reply:
Oh good point. It does make it slightly more annoying to add formatter specific methods but I think it's fine.